src/components/erc721/modals/search-modal.tsx

```diff
@@ -0,0 +1,209 @@
+}
+
+const ModalContent = styled.div`
+    overflow: auto;
+
+    @media (min-width: ${themeBreakPoints.md}) {
+        height: 500px;
+    }
+`;
+
+const SearchStyled = styled(InputSearch)`
+    background-color: ${props => props.theme.componentsTheme.modalSearchFieldBackgroundColor};
+    border-color: ${props => props.theme.componentsTheme.modalSearchFieldBackgroundColor};
+    color: ${props => props.theme.componentsTheme.modalSearchFieldTextColor};
+    font-size: 16px;
+    font-weight: 600;
+    outline: none;
+    width: 100%;
+
+    ::placeholder {
+        color: ${props => props.theme.componentsTheme.modalSearchFieldPlaceholderColor};
+        font-weight: 400;
+        text-align: left;
+    }
+`;
+
+const FiltersMenu = styled.div`
+    border-top: 1px solid ${props => props.theme.componentsTheme.borderColor};
+    display: flex;
+    flex-direction: column;
+    margin: 0 auto 40px;
+    max-width: ${themeBreakPoints.xxl};
+    padding: 30px 0 0 0;
+    position: relative;
+    width: 100%;
+    z-index: 1;
+
+    @media (min-width: ${themeBreakPoints.md}) {
+        align-items: start;
+        flex-direction: row;
+    }
+`;
+
+const CollectiblesListCount = styled.p`
+    color: ${props => props.theme.componentsTheme.textColorCommon};
+    flex-grow: 1;
+    font-size: 16px;
+    line-height: 1.2;
+    margin: 0 0 25px;
+    padding: 0;
+    text-align: left;
+
+    @media (min-width: ${themeBreakPoints.md}) {
+        margin-bottom: 0;
+    }
+`;
+
+const CollectiblesFilterDropdown = css`
+    font-size: 14px;
+    margin-bottom: 25px;
+    margin-right: auto;
+    position: relative;
+
+    @media (max-width: ${themeBreakPoints.sm}) {
+        > div:nth-child(2) {
+            left: 0;
+            right: auto;
+        }
+    }
+
+    @media (min-width: ${themeBreakPoints.md}) {
+        margin-bottom: 0;
+        margin-right: 25px;
+    }
+
+    &:last-child {
+        margin-bottom: 0;
+
+        @media (min-width: ${themeBreakPoints.md}) {
+            margin-right: 0;
+        }
+    }
+`;
+
+const CollectiblesListSortStyled = styled(CollectiblesListSort)`
+    ${CollectiblesFilterDropdown}
+    z-index: 5;
+`;
+const CollectiblesListFilterStyled = styled(CollectiblesListFilter)`
+    ${CollectiblesFilterDropdown}
+    z-index: 1;
+`;
+
+const initialState = {
+    filterType: CollectibleFilterType.ShowAll,
+    searchText: '',
+    sortType: CollectibleSortType.NewestAdded,
+};
+
+class SearchModal extends React.Component<Props, State> {
+    public state = { ...initialState };
+
+    public render = () => {
+        const { isOpen, theme, allCollectibles } = this.props;
+        const { searchText, sortType, filterType } = this.state;
+
+        const collectibles = Object.keys(allCollectibles).map(key => allCollectibles[key]);
+        const searchResult = filterCollectibleByName(collectibles, searchText);
+        const shouldShowResults = searchText.length > 0;
+
+        Object.assign(theme.modalTheme.content, {
```

Comment:
Req: This is modifying something that it's received via props. I think this may be the reason why the other modals look wider now.